docs/source/quickstart.rst

```diff
@@ -72,10 +76,16 @@ rasterio
 Supports any file format supported by ``rasterio.open`` - most commonly
 geotiffs.
 
+Note: Consider installing ``rioxarray`` and using the ``netcdf`` driver with ``engine="rasterio"``.
```

Comment:
should we recommend this path for `tif`? or rather `intake.open_rasterio`

Reply:
Now that either is possible, I don't know. Have you had any different experience with the two?

Reply:
agreed